sec1_intro_and_setup/test_app/config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```

Comment:
Whats this file for?

Reply:
`config/database.yml` allows you to specify all the information needed to access your database. By default, there are 3 environment run by Rails: development, test, and production.